released/packages/coq-paramcoq/coq-paramcoq.1.1.2+coq8.10/descr

```diff
@@ -0,0 +1,5 @@
+Paramcoq
```

Comment:
The content of this file should be in the `description` field of the `opam` file, as of opam 2.

Reply:
Thanks, done